src/main/java/com/google/sps/dao/GeolocationDao.java

```diff
@@ -0,0 +1,24 @@
+package com.google.sps.dao;
+
+import com.google.sps.Mock;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+
+public class GeolocationDao {
+
+  public static Map<String, String> ipCountryMap;
```

Comment:
Avoid having static functions and entries, that is generally not a good practice.

Reply:
Fixed, integrated into DAO mock